Project.toml

```diff
@@ -1,7 +1,7 @@
 name = "FeatureDescriptors"
 uuid = "cb0561c3-42a1-46a3-8749-4ae46de0afcc"
 authors = ["Invenia Technical Computing Corporation"]
-version = "1.1.0"
+version = "1.2.0"
```

Comment:
is this a breaking change? since it adds new requirements to the test interface?

Reply:
It does add a new requirement to the test interface, but our default behaviour for `parents` should mean that the new test passes by default. I'll run the internally dependencies using this change to confirm though.

Reply:
Okay, ran this on our internal codebase and the only issue I ran into is that the `union` in the test doesn't work if `sources`/`parents` is empty.